doc/rendszerterv.md

```diff
@@ -9,19 +9,82 @@ Az már nem cél, hogy Androidos, vagy IOS eszközön fusson, mert arra külön
  A megoldásra a felhasználó pontszámot kap.
 ---
 ## 2. Projektterv
+### Projektszerepkörök, felelősségek:
+* Scrum master: Varga József-Márk
+* Product owner: Varga József-Márk
 
+### Projektmunkások és felelőségek:
+
+* Backend munkálatok:
+    * Varga József-Márk, Bódi András, Cserés Gábor Feladatuk a funkciók létrehozása illetve adatok tárolása
+
+* Frontend:
+    * Varga József-Márk, Bódi András, Cserés Gábor Feladatuk a weboldal megjelenítése
+
+### Ütemterv:
+
+|          Funkció        |Feladat|Prioritás|Becslés|Aktuális becslés|Eltelt idő|Hátralévő idő|
+|          :----:         |:----: |  :----: | :----:|     :----:     |  :----:  |    :----:   |
+|Követelmény specifikáció |       |0	    |7      |7               |7		|       0     |
+|Funkcionális specifikáció| 	  |0	    |7	    |7		     |7		|	0     |
+|       Rendszerterv      | 	  |0        |7      |7		     |7		|	0     |
+|         Frontend        | 	  |2   	    |7      |7	       	     |0		|	7     |
+|         Backend         | 	  |2	    |7      |7		     |0		|	7     |
 ---
 ## 3. Üzleti folyamatok modellje
 
 ---
 ## 4. Követelmények
-
+* Funkcionális követelmények:
+    * Felhasználó tudja a szavakat mozgatni, hogy az angol szót a magyar megfelelőjével tudja párosítani
+    * Felhasználó csak a kijelölt helyre tudja mozgatni a szavakat
+    * Egy gomb segítségével tudja ellenőrizni a felhasználó, a megoldásait
+    * Ellenőrző gomb lenyomásával pedig az oldal meg mondja, hogy a megoldás hibátlan vagy hibás
+    * Egy másik gombbal pedig előröl tudja kezdeni a szavak húzgálását
+    * Legyen egy külön felület ahol angol-magyar szavakat lehet a rendszerbe bevinni még
+    * Lehessen szavakat kivenni
+    * A weboldal a legtöbb böngészőt támogassa
+    * A weboldalt lehessen telefonon is használni, illetve különböző képernyőkhöz tudjon igazodni
```

Comment:
Az ,,illetve kulonbozo ... ,, reszt lehet ki kene venni mert eleg macera egy reszponziv oldalt csinalni ami telon es minden eszkozon is jo